src/libmongoc/examples/client-side-encryption-helpers.c

```diff
@@ -13,8 +13,9 @@ hex_to_bin (const char *hex, uint32_t *len)
       return NULL;
    }
 
-   if (!bson_in_range_unsigned (uint32_t, hex_len / 2u)) {
-      return NULL;
+   size_t num_bytes = hex_len / 2;
+   if ((uint64_t) num_bytes >= UINT32_MAX) {
```

Comment:
```suggestion
   size_t num_bytes = hex_len / 2u;
   if (num_bytes >= UINT32_MAX) {
```
Consistent signedness -> cast not required.